File: 프로그래머스/1/12928. 약수의 합/약수의 합.java
import java.util.*;
class Solution {
    public int solution(int n) {
        return Arrays.stream(get(n)).sum();
    }
    
    public int[] get(int n) {
        List<Integer> l = new ArrayList<>();
        for (int i = 1; i <= n; i++) {
            if (n % i == 0) l.add(i);
        }
        return l.stream().mapToInt(Integer::intValue).toArray();
    }
}

File: 프로그래머스/1/12928. 약수의 합/약수의 합.kt
class Solution {
    fun solution(n: Int): Int {
        return c(n).sum()
    }
    
    fun c(n: Int): Array<Int> {
        val list = mutableListOf<Int>()
        for (i in 1..n) {
            if (n % i == 0) list.add(i)
        }
        return list.toTypedArray()
    }
}